spec/factories/capital_summary_factory.rb

```diff
@@ -0,0 +1,31 @@
+FactoryBot.define do
+  factory :capital_summary do
+    assessment
+
+    trait :pending do
+      capital_assessment_result { 'pending' }
+    end
+
+    trait 'summarised' do
+      capital_assessment_result { 'summarised' }
+      lower_threshold { 3_000 }
+      upper_threshold { 8_000 }
+    end
+
+    trait :below_lower_threshold do
+      total_liquid { 350 }
+      total_capital { 350 }
+      assessed_capital { 350 }
+    end
+
+    trait :at_lower_threshold do
+      assessed_capital { 3_000 }
+    end
+
+    trait :above_lower_threshold do
+      total_liquid { 3_000.01 }
+      total_capital { 3_000.01 }
+      assessed_capital { 3_000.01 }
+    end
```

Comment:
Again use the current thresholds and then:
```ruby
Faker::Number.between(from: 1, to: (threshold - 1)
```
and
```ruby
Faker::Number.between(from: (threshold + 1, to: (2 * threshold))
```
Or something similar so the numbers aren't hard coded, and random numbers are used to avoid the test being run on a number that just happens to be a special case.